deliverylocation/admin.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib import admin
+from deliverylocation.models import DeliveryLocation, Supplier
```

Comment:
I think the package name should be `delivery_location` rather than `deliverylocation`.

Reply:
Done!